target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -457,6 +457,11 @@ tplink,tl-wa801nd-v5)
 	ucidef_set_led_wlan "wlan" "wlan" "$boardname:green:wlan" "phy0tpt"
 	ucidef_set_led_netdev "lan" "lan" "$boardname:green:lan" "eth0"
 	;;
+tplink,tl-mr3020-v3)
+	set_usb_led "tp-link:green:3g"
+	set_wifi_led "tp-link:green:wlan"
```

Comment:
Why don't you set wlan and 3g leds in DTS?

Reply:
because it isn't yet used in ramips. all existing dts files should be converted to "set trigger via dts" first. I'm already working on [doing it for the usb trigger](https://git.openwrt.org/?p=openwrt/staging/mkresin.git;a=shortlog;h=refs/heads/ramips-trigger-cleanup).